exp/api/v1beta1/gcpmanagedcontrolplane_types.go

```diff
@@ -29,20 +29,23 @@ const (
 
 // GCPManagedControlPlaneSpec defines the desired state of GCPManagedControlPlane.
 type GCPManagedControlPlaneSpec struct {
-	// EnableAutopilot indicates whether to enable autopilot for this GKE cluster.
-	EnableAutopilot bool `json:"enableAutopilot"`
+	// Project is the name of the project to deploy the cluster to.
+	Project string `json:"project"`
 	// Location represents the location (region or zone) in which the GKE cluster
 	// will be created.
 	Location string `json:"location"`
+	// EnableAutopilot indicates whether to enable autopilot for this GKE cluster.
+	EnableAutopilot bool `json:"enableAutopilot"`
 	// ReleaseChannel represents the release channel of the GKE cluster.
 	// +optional
-	ReleaseChannel *string `json:"releaseChannel,omitempty"`
+	ReleaseChannel *ReleaseChannel `json:"releaseChannel,omitempty"`
```

Comment:
Whats the default for this? `stable`?

Reply:
i think it's `regular`

Reply:
updated the comment with that information